test-projects/access-control/cypress/integration/list/admin-ui.js

```diff
@@ -159,13 +159,13 @@ describe('Access Control Lists > Admin UI', () => {
           const prettyName = prettyListName(name);
           const slug = listSlug(name);
 
-          // Still navigable
-          cy.get('body nav').should('contain', prettyName);
+          // Not navigable
```

Comment:
I think this test was backwards. Its description said "does **not** show items when readable & not admin" but it was checking that the sidebar did contain it. Changed to not contain.